gitclone.adoc

```diff
@@ -9,18 +9,15 @@
 ////
 == Getting started
 
-The fastest way to work through this guide is to clone its Git repository and use the provided projects
-inside. To do this, run the following commands:
+Run the following command to clone the Git repository and use the projects provided
+inside:
 
 [subs="attributes"]
 ----
 git clone https://github.com/openliberty/guide-{projectid}.git
 cd guide-{projectid}
 ----
 
-The `start` directory contains the starting project for this guide. Use this directory as the starting
-point for the guide.
+The `start` directory contains the starting project that you can use to begin.
 
-The `finish` directory contains the finished project for this guide. This is what you'll end up with
-once you have worked through the guide. Feel free to run whats in this directory to give yourself a
-better idea of what you will be building.
+The `finish` directory contains the finished project. To see what you'll build, run the commands in this directory.
```

Comment:
I agree with YK, having "what you'll build" in this paragraph overlaps a bit with the next section which is called "Try what you'll build". Maybe we can instead say "The `finish` directory contains the finished project. This is what you will be building."

Reply:
@sishida I like that, can you add that to your pull request?